nilearn/plotting/surf_plotting.py

```diff
@@ -0,0 +1,376 @@
+"""
+Functions for surface visualization.
+Only matplotlib is required.
+"""
+
+from nilearn._utils.compat import _basestring
```

Comment:
We should probably use relative imports here: "from .._utils.compat import _basestring"

Also, these imports should be moved to the end of the list of imports, as imports should be ordered from general (standard library) to local.